### PR TITLE
Allow having a domain as path parameter

### PR DESCRIPTION
**Pull request by amertkara**
JIRA ticket: https://appdirect.jira.com/browse/PI-12150

We needed to configure Spring to stop truncating domains in the URL.

- [x] Approved by a PI tech lead

@hooriak @davidchau

### CODE REVIEW

**Comment by davidchau** [comment]
Please make sure it's working with GoDaddy connector

**Comment by amertkara** [comment]
@davidchau Good point

I tested with the chatty-pie connector, will run the change with GoDaddy and Google connectors

**Comment by hooriak** [comment]
Ready to merge once we have integration tests.

File: src/test/java/com/appdirect/sdk/appmarket/domain/DomainOwnershipControllerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import com.appdirect.sdk.support.FakeAppmarket;
+import wiremock.org.apache.commons.lang3.RandomStringUtils;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = FullConnector.class, webEnvironment = RANDOM_PORT)
+public class DomainOwnershipControllerIntegrationTest {
+	@LocalServerPort
+	private int localConnectorPort;
+	@Autowired
+	public DomainRemovalHandler domainRemovalHandler;
```
Comment (hooriak) [inline src/test/java/com/appdirect/sdk/appmarket/domain/DomainOwnershipControllerIntegrationTest.java]: why is this public?

Comment (amertkara) [reply src/test/java/com/appdirect/sdk/appmarket/domain/DomainOwnershipControllerIntegrationTest.java]: must have slipped, will fix that

**Comment by amertkara** [comment]
### Google connector tests PASSED with the change
screen shot 2018-01-19 at 1 14 38 pm

### Godaddy connector tests PASSED with the change
screen shot 2018-01-19 at 1 04 46 pm

**Comment by hooriak** [comment]
merging..

**Pull request merged**